scripts/machine_translation/README.md

```diff
@@ -55,10 +71,10 @@ python evaluate_transformer.py \
 
 Test BLEU score with 3 seeds (evaluated via sacre BLEU):
 
-- transformer_base
+- transformer_wmt_en_de_big
```

Comment:
Try not to remove the transformer_base results.

Reply:
transformer base results will be added after tuning and training